### ISSUE TITLE
Set Password dialog box in OpenOffice cannot be read

### ISSUE THREAD

**Issue by xtstoll** [nvaccess/nvda#7505]
### Steps to reproduce:
1. Launch OpenOffice
2. Start a new document
3. Save new document and enable "Save with password" option

### Expected behavior:
NVDA should be able to read this dialog box.

### Actual behavior:
NVDA says nothing.

### System configuration:
NVDA installed, version 2017.2 in Brazilian Portuguese
Windows 7 in Brazilian Portuguese
OpenOffice 4.1.3 in Brazilian Portuguese

Screenshot of the dialog box:
![libreoffice-save-as-wpassword](https://user-images.githubusercontent.com/24814666/29488606-b77c075e-84e4-11e7-96e6-c144d317c588.png)

**Comment by Brian1Gaff** [comment]
Just a suggestion, originally this program required Java to work. It was 
modified so it apparently did not need it, but maybe some parts still do and 
need the access bridge.
 I've not seen much change on Open Office for some time, unlike Libra 
Office, are there still people working on it?
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.

**Comment by zahra21** [comment]
hi.
no, both libreoffice and openoffice dont need java or java access briedge for normal use.
openoffice supports iaccessible2 since version 4.1.0 and libreoffice also supports it in its version since 4.3.0, both in 2014!
libreoffice i think need java only for base and its more independent from java.
but openoffice need java for more tasks like creating templet letter, fax, agenda, if i am correct, also save as xml and creating web page.
yes, openoffice currently updated one version in a year.
in previous weeks, i read that openoffice 4.1.4 rc2 is available for download.i think the final version becomes released in the next month.

attention!
openoffice 4.1.4 and 4.1.5 are both available for download, but the changes are some bug fixes only and these versions have not new features.
dont use 4.1.4, because selecting entire document and copying it, causes openoffice 4.1.4 to crash!
this issue is resolved in 4.1.5
i think at least, in writer, libreoffice should be more independent from java and java access briedge.

**Comment by zahra21** [comment]
hi again.
i updated my previous comments and wrote some important informations.
please read my previous comment since the line which i wrote:
attention!
also, i tested different versions of libreoffice and did not have any issues in reading dialog of saving documents with password!
i just tested different versions of libreoffice on xp and not other windows versions, and with different versions of nvda.
nvda says everything without any problems for saving documents with password!
did you try this issue with libreoffice?
libreoffice is frequently updated and i think is more accessible in some cases these days!

**Comment by Adriani90** [comment]
@xtstoll are you still having this issue?

**Comment by xtstoll** [comment]
Just retested, and it's OK now. I could not test with the same exact conditons, but I think it's OK. Working in the lastest NVDA.

NVDA installed, version 2019.1 in Brazilian Portuguese
Windows 10 in Brazilian Portuguese
OpenOffice 4.1.5 in Brazilian Portuguese

Screenshot of dialog "Set Password" tested:
<img width="264" alt="soffice bin_2019-04-05_14-40-55" src="https://user-images.githubusercontent.com/24814666/55646312-3d40de00-57b1-11e9-9a48-d9cc75ae293d.png">
